src/faebryk/core/cpp/__init__.py

```diff
@@ -117,7 +117,9 @@ def compile_and_load():
                 is_pyi=True,
             )
         )
-        run_live(["ruff", "check", "--fix", pyi_out], logger=logger)
+        run_live(
```

Comment:
This fixes a bug that was causing the formatter to fail on my machine. I think it was using the global python for some reason?